docker/Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: ci
```

Comment:
FYI There is also a make file inside the `udb3-vagrant` root folder. But it seems good to add the docker commands to this makefile.

Reply:
Yeah, I think it makes more sense to have a separate Makefile for Docker related commands.